samples/api-client/accesstoken_example/accesstoken.py

```diff
@@ -0,0 +1,177 @@
+    # algorithm = 'RS256'
+    # certificate_file = 'path/to/certificate.pem'
+
+    jwt_token = generate_iot_jwt_token(project_id, algorithm, certificate_file)
+    token = exchange_iot_jwt_token_with_gcp_token(
+        cloud_region, project_id, registry_id, device_id, jwt_token, scope
+    )
+    return token
+    # [END iot_generate_gcp_token]
+
+
+def generate_iot_jwt_token(project_id, algorithm, path_to_private_certificate):
+    """Generate cloud iot jwt token."""
+    # [START iot_generate_iot_jwt_token]
+    # project_id = 'YOUR_PROJECT_ID'
+    # algorithm = 'RS256'
+    # certificate_file = 'path/to/certificate.pem'
+    jwt_payload = '{{"iat":{},"exp":{},"aud":"{}"}}'.format(
+        time.time(),
+        time.mktime((datetime.now() + timedelta(hours=6)).timetuple()),
+        project_id,
+    )
+    private_key_bytes = ""
+    with io.open(path_to_private_certificate) as f:
+        private_key_bytes = f.read()
+    encoded_jwt = jwt.encode(
+        json.loads(jwt_payload), private_key_bytes, algorithm=algorithm
+    )
+    return encoded_jwt
+    # [END iot_generate_iot_jwt_token]
+
+
+def exchange_iot_jwt_token_with_gcp_token(
+    cloud_region, project_id, registry_id, device_id, jwt_token, scopes
+):
+    """Exchange iot jwt token for gcp token."""
+    # [START iot_exchange_iot_jwt_token_with_gcp_token]
+    # cloud_region = 'us-central1'
+    # project_id = 'YOUR_PROJECT_ID'
+    # registry_id = 'your-registry-id'
+    # device_id = 'your-device-id'
+    # jwt_token = 'CLOUD_IOT_GENERATE_JWT_TOKEN'
+    # scopes = 'scope1 scope2' https://developers.google.com/identity/protocols/oauth2/scopes
+    global HOST
+    resource_url = "projects/{}/locations/{}/registries/{}/devices/{}".format(project_id, cloud_region, registry_id, device_id)
+    request_path = "{}/v1alpha1/{}:generateAccessToken".format(
+        HOST, resource_url
+    )
+    headers = {"authorization": "Bearer {}".format(jwt_token)}
+    request_payload = {"scope": scopes, "device": resource_url}
+    resp = req.post(url=request_path, data=request_payload, headers=headers)
+    print(resp.raise_for_status())
+    return resp.json()
+    # [END iot_exchange_iot_jwt_token_with_gcp_token]
+
+
+def parse_command_line_args():
+    """Parse command line arguments."""
+    parser = argparse.ArgumentParser(
+        description=__doc__, formatter_class=argparse.RawDescriptionHelpFormatter
+    )
+
+    parser.add_argument(
+        "--algorithm",
+        default="RS256",
+        choices=("RS256", "ES256"),
+        help="Which encryption algorithm to use to generate the JWT.",
+    )
+    parser.add_argument("--certificate_path", help="Path to private certificate.")
+    parser.add_argument(
+        "--cloud_region", default="us-central1", help="GCP cloud region"
+    )
+
+    parser.add_argument("--device_id", default=None, help="Device id.")
+    parser.add_argument(
+        "--scope", default=None, help="Scope for GCP token. Space delimited strings"
```

Comment:
add `See the full list of scopes at: https://developers.google.com/identity/protocols/oauth2/scopes` in the help text